content/en/docs/zero-code/python/_index.md

```diff
@@ -37,7 +38,10 @@ packages installed in your active `site-packages` folder, and installs the
 corresponding instrumentation libraries for these packages, if applicable. For
 example, if you already installed the `flask` package, running
 `opentelemetry-bootstrap -a install` will install
-`opentelemetry-instrumentation-flask` for you.
+`opentelemetry-instrumentation-flask` for you. The OpenTelemetry Python agent
+will use
+[monkey patching](https://docs.pytest.org/en/stable/how-to/monkeypatch.html#monkeypatching-functions)
```

Comment:
I would like to wikipedia article about monkey patching instead. We are not using `monkeypatch`.

Reply:
Also, I would add the reference to the first time where it was mentioned